polartools/load_data.py

```diff
@@ -232,12 +232,12 @@ def is_Bluesky_specfile(spec_file, folder=''):
     --------
     :func:`spec2nexus.spec.SpecDataFile`
     """
-    if isinstance(spec_file, str):
-        path = join(folder, spec_file)
+    if isinstance(source, str):
+        path = join(folder, source)
         source = SpecDataFile(path)
 
     for comment in source.headers[0].comments:
-        if 'Bluesky' in comment:
+        if "Bluesky" in comment:
```

Comment:
A stronger verification would check only the first comment.

```python
    if len(source.headers[0].comments):
        return source.headers[0].comments[0].startswith("Bluesky")
```

Reply:
> A stronger verification would check only the first comment.
> 
> ```python
>     if len(source.headers[0].comments):
>         return source.headers[0].comments[0].startswith("Bluesky")
> ```

Very elegant. I implemented it and it seems to work.